src/popup/router/components/InputField.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div class="input-field">
+    <label class="label" v-if="label">
+      {{ label }}
+    </label>
+    <div :class="{ isFocused, error }" class="container">
+      <slot v-if="!error" name="left" />
+      <Error v-else />
+      <input
+        :type="type"
+        class="input"
+        :placeholder="placeholder"
+        :value="value"
+        @input="$emit('input', $event.target.value)"
+        @focus="isFocused = true"
+        @blur="isFocused = false"
+        :data-cy="type ? `input-${type}` : 'input'"
+        :disabled="disabled"
+      />
+      <slot name="right" />
+    </div>
+  </div>
+</template>
+
+<script>
+import Error from '../../../icons/error.svg?vue-component';
+
+export default {
+  components: { Error },
+  props: {
+    value: [String, Number],
+    error: Boolean,
+    placeholder: String,
+    type: {
+      type: String,
+      default: 'text',
+    },
+    label: String,
+    disabled: Boolean,
+  },
+  data: () => ({
+    isFocused: false,
+  }),
+};
+</script>
+
+<style lang="scss" scoped>
+@import '../../../styles/typography';
+
+.input-field {
+  .label {
+    margin: 8px 0;
+    display: block;
+
+    @extend %face-sans-15-medium;
+
+    color: $color-dark-grey;
+    text-align: left;
+  }
+
+  .container {
+    display: flex;
+    align-items: center;
+    padding: 8px 16px;
+    height: 40px;
+    background-color: $color-bg-2;
+    border: 1px solid transparent;
+    border-radius: 6px;
+
+    &.isFocused {
```

Comment:
```suggestion
    &:focus-within {
```
🚀

Reply:
@etharner didn't it work for some reason?
https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within